README.md

```diff
@@ -55,6 +56,27 @@ Note: grab-site currently **does not work with Python 3.5**; please use Python 3
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
+Install with Docker
+---
+Get the pre-built docker container:
```

Comment:
I would really like the instructions to be so complete that people who don't have Docker installed yet (me included) know where to get it.